Prog/NCHousing_needs_units_targets.sas

```diff
@@ -0,0 +1,927 @@
+totalpop=0.2;
+merge=1;
+totpop_wt= totalpop*AFACT2; 
+geoid=.;
+if county2_char= "0100" then geoid=1;
+else if  county2_char= "0200" then geoid=2;
+else if  county2_char= "0300" then geoid=3;
+else if  county2_char= "0400" then geoid=4;
+else if  county2_char= "0500 or 0600" then geoid=5;
+else if  county2_char= "0700" then geoid=6;
+else if  county2_char= "0800" then geoid=7;
+else if  county2_char= "0900" then geoid=8;
+else if  county2_char= "1000" then geoid=9;
+else if  county2_char= "1100" then geoid=10;
+else if  county2_char= "1201 to 1208" then geoid=11;
+else if  county2_char= "1301 to 1302" then geoid=12;
+else if  county2_char= "1400" then geoid=13;
+else if  county2_char= "1500" then geoid=14;
+else if  county2_char= "1600" then geoid=15;
+else if  county2_char= "1701 to 1704" then geoid=16;
+else if  county2_char= "1801 to 1803" then geoid=17;
+else if  county2_char= "1900 or 2900" then geoid=18;
+else if  county2_char= "2000" then geoid=19;
+else if  county2_char= "2100" then geoid=20;
+else if  county2_char= "2201 to 2202" then geoid=21;
+else if  county2_char= "2300 or 2400" then geoid=22;
+else if  county2_char= "2500" then geoid=23;
+else if  county2_char= "2600 or 2700" then geoid=24;
+else if  county2_char= "2800" then geoid=25;
+else if  county2_char= "3001 to 3003" then geoid=26;
+else if  county2_char= "3101 to 3108" then geoid=27;
+else if  county2_char= "3200 or 3300" then geoid=28;
+else if  county2_char= "3400" then geoid=29;
+else if  county2_char= "3500" then geoid=30;
+else if  county2_char= "3600" then geoid=31;
+else if  county2_char= "3700" then geoid=32;
+else if  county2_char= "3800" then geoid=33;
+else if  county2_char= "3900" then geoid=34;
+else if  county2_char= "4000" then geoid=35;
+else if  county2_char= "4100 or 4500" then geoid=36;
+else if  county2_char= "4200" then geoid=37;
+else if  county2_char= "4300" then geoid=38;
+else if  county2_char= "4400" then geoid=39;
+else if  county2_char= "4600 or 4700" then geoid=40;
+else if  county2_char= "4800" then geoid=41;
+else if  county2_char= "4900 or 5100" then geoid=42;
+else if  county2_char= "5001 to 5003" then geoid=43;
+else if  county2_char= "5200" then geoid=44;
+else if  county2_char= "5300 or 5400" then geoid=45;
+
+run;
+proc sort data= fiveyeartotal1;
+by geoid;
+run;
+proc summary data=fiveyeartotal1;
+by geoid;
+var totpop_wt;
+weight hhwt;
+output out=geo_sum sum=ACS_13_17;
+run; 
+proc sort data= projectedHH;
+by geoid;
+run;
+
+data calculate_calibration;
+merge geo_sum(in=a) projectedHH;
+by geoid;
+if a;
+calibration=(hh2015/ACS_13_17);
+run;
+
+data fiveyeartotal_c;
+merge fiveyeartotal1 calculate_calibration;
+by geoid;
+
+hhwt_geo=.; 
+
+hhwt_geo=hhwt*calibration*0.2; 
+
+label hhwt_geo="Household Weight Calibrated to Steven Estimates for Households"
+	  calibration="Ratio of Steven 2015 estimate to ACS 2013-17 for 45 geographic units";
+
+run; 
+
+proc univariate data= fiveyeartotal_c;
```

Comment:
does this result match the one from the household income projections program?

Reply:
that one is not calibrated, and this is calibrated using uncalibrated tabulation and projection outcome. @lhendey

Reply:
@maggiesu0725 i'm not following your response.